parse/table_definitions_avalanche/aave/AToken_v2_event_Approval.json

```diff
@@ -0,0 +1,54 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "internalType": "address",
+                    "name": "owner",
+                    "type": "address"
+                },
+                {
+                    "indexed": true,
+                    "internalType": "address",
+                    "name": "spender",
+                    "type": "address"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "value",
+                    "type": "uint256"
+                }
+            ],
+            "name": "Approval",
+            "type": "event"
+        },
+        "contract_address": "SELECT aToken FROM ref('LendingPoolConfigurator_v2_event_ReserveInitialized')",
```

Comment:
Hi for this kind of reference, the referred event (here `LendingPoolConfigurator_v2_event_ReserveInitialized`) needs to be submitted through a separate PR and merged first, then we can merge the current PR, so the tables can be built in order. Can you pls update thanks!

Reply:
@daishuai323 OK sounds good. I made the PR for LendingPoolConfigurator_v2 events here: https://github.com/nansen-ai/evmchain-etl-table-definitions/pull/119